ledger/sandbox/src/main/resources/db/migration/V1__Init.sql

```diff
@@ -125,10 +121,12 @@ CREATE TABLE contract_key_maintainers (
 CREATE UNIQUE INDEX contract_key_maintainers_idx
   ON contract_key_maintainers (contract_id, maintainer);
 
--- a generic table to store meta information such as: ledger id and ledger end
+-- this table is meant to have a single row storing all the
```

Comment:
all the...?

Reply:
haha, I must have had an interrupt to leave this cliffhanger here.. will fix it